lib/usd/schemas/CMakeLists.txt

```diff
@@ -180,7 +180,7 @@ target_link_libraries(${TARGET_PYTHON_NAME}
 # run-time search paths
 # -----------------------------------------------------------------------------
 if(IS_MACOSX OR IS_LINUX)
-    mayaUsd_init_rpath(rpath "${PROJECT_NAME}")
+    mayaUsd_init_rpath(rpath mayaUsd)
```

Comment:
Since we restructured lib sub-directories, ${PROJECT_NAME} no longer returns "mayaUsd" and rather "maya-usd" which would be wrong.   I can make a global variable at the super project level.  I also don't like the hard-coded names.

Reply:
@ppt-adsk  
Please see more info on this here:
https://github.com/Autodesk/maya-usd/pull/389#discussion_r399602894